MAUI/DataGrid/selection.md

```diff
@@ -122,6 +122,34 @@ public partial class MainPage : ContentPage
 
 ![All the rows selected in .NET MAUI DataGrid](Images/selection/maui-datagrid-selectall.png)
 
+## Column selection
+The SfDataGrid provides the [SfDataGrid.AllowColumnSelection]() property to select columns when the `SelectionMode` is set to Multiple, the `SelectionUnit` is set to Cell, and the `NavigationMode` is set to Cell.
+
+By default, the `SortingGestureType` is set to `Tap` mode, so we select the column by double-clicking the header cell. We can also modify this behavior by changing the `SortingGestureType`.
```

Comment:
The `SfDataGrid` includes the [SfDataGrid.AllowColumnSelection](...) property, which allows you to select columns by tapping the column header cell. This feature is enabled only when the `SelectionMode` is set to `Multiple`, the `SelectionUnit` is set to `Cell`, and the `NavigationMode` is set to `Cell`.

**Note:** When the `SortingGestureType` is set to `Tap`, column selection is applied only when double-tapping the column header cell. This ensures both sorting and column selection interactions can occur without conflicts.

Reply:
modified